src/gui/key/backup.sh

```diff
@@ -0,0 +1,43 @@
+gui_key_backup(){
+    details=$(yad --title="EasyGnuPG | Key Backup" \
```

Comment:
I think that `details` should be declared as `local`. In some cases this does not make any difference, but it is a good habit and a good practice to declare local variables as `local`.